scripts/validate_version_bump.py

```diff
@@ -18,7 +18,7 @@ def main():
         raise ValueError(f"Couldn't extract version from {version_path}") from e
 
     try:
-        with popen(f"git show main:src/citrine/__version__.py", mode="r") as fh:
+        with popen(f"git show origin/main:src/citrine/__version__.py", mode="r") as fh:
```

Comment:
Was this a requirement to get actions to run?  Seems weird you'd refer to a remote branch, especially if you don't precede it with a fetch.

Reply:
Yes. I think the fetch would only be needed for the local branch, right? And referring to the remote will check the actual live value.

Reply:
I did end up adding `fetch`, because sometimes it worked and sometimes it didn't. 🤷